test/fixtures/product_fixtures.js

```diff
@@ -14,12 +14,41 @@ module.exports = {
     pactProducts.pactify(opts)
   },
 
+  validUpdateProductServiceNameRequest: (serviceName) => {
+    const data = {
+      op: 'update',
+      path: 'service_name',
+      value: serviceName || 'New Service Name'
+    }
+    return {
+      getPactified: () => {
+        return pactProducts.pactify(data)
+      },
+      getPlain: () => data
+    }
+  },
+
+  invalidUpdateProductServiceNameRequest: () => {
```

Comment:
its not really productServiceName :(. Its more of a gateway account name we are updating